api/src/routers/future-meal.js

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const futureMealroute = express.Router();
+
+futureMealroute.get("/", async (req, res, next) => {
+    console.log("Received request for /api/all-meal");
+    try{
+        const futureMeal_meals = await knex("meal").where("when",">",new Date()).orderBy("id");
```

Comment:
The Date time field requires a special representation in SQL. You can explore knex helpers like https://knexjs.org/guide/utility.html#now to solve these common issues

```suggestion
        const futureMeal_meals = await knex("meal").where("when",">",knex.fn.now()).orderBy("id");
```

Reply:
thank you for feeedback , made changes accordingly